float64 acc_x
float64 acc_y
float64 theta_acc
bool stay_still
